### PR TITLE
Update vagrant to k8s 1.6 and etcdv3

### PR DESCRIPTION
**Pull request by djosborne**

### CODE REVIEW

**Comment by tomdee** [comment]
So bind the times - rc1 is already out https://github.com/kubernetes/kubernetes/releases/tag/v1.6.0-rc.1

**Comment by caseydavenport** [comment]
v1.6.0 was released today.

File: master/getting-started/kubernetes/installation/vagrant/master-config.yaml [hunk 1]
```diff
@@ -40,6 +40,7 @@ coreos:
         --runtime-config=extensions/v1beta1=true,extensions/v1beta1/networkpolicies=true \
         --insecure-bind-address=0.0.0.0 \
         --admission-control=NamespaceLifecycle,LimitRanger,ServiceAccount,ResourceQuota \
+        --storage-backend etcd2 \
```
Comment (caseydavenport) [inline master/getting-started/kubernetes/installation/vagrant/master-config.yaml]: Let's upgrade our Vagrant demo to use etcdv3.  This is really important.

File: v2.1/getting-started/kubernetes/installation/vagrant/index.md [hunk 2]
```diff
@@ -1,6 +1,5 @@
 ---
 title: Deploying Calico and Kubernetes on Container Linux by CoreOS using Vagrant and VirtualBox
-redirect_from: latest/getting-started/kubernetes/installation/vagrant/index
```
Comment (caseydavenport) [inline v2.1/getting-started/kubernetes/installation/vagrant/index.md]: What's this change for?

Comment (djosborne) [inline v2.1/getting-started/kubernetes/installation/vagrant/index.md]: from fat fingers. good catch.

**Pull request merged**